compiler-plugin/src/main/java/io/ballerina/stdlib/graphql/compiler/schema/generator/GraphqlSourceModifier.java

```diff
@@ -445,10 +443,13 @@ private SeparatedNodeList<MappingFieldNode> getUpdatedFields(MappingConstructorE
         List<Node> fields = new ArrayList<>();
         SeparatedNodeList<MappingFieldNode> existingFields = annotationValue.fields();
         Token separator = NodeFactory.createToken(SyntaxKind.COMMA_TOKEN);
-        for (MappingFieldNode field : existingFields) {
-            fields.add(field);
-            fields.add(separator);
+        for (int i = 0; i < existingFields.size(); i++) {
+            fields.add(existingFields.get(i));
+            if (i < existingFields.size() - 1) {
```

Comment:
I think, if the seize is 2, we need to add a separator in here.

Reply:
Sorry, I did not understand this point.